src/Microsoft.VisualStudio.ProjectSystem.Managed.VS.UnitTests/ProjectSystem/VS/RenamerTests.cs

```diff
@@ -5,27 +5,54 @@
 using Microsoft.CodeAnalysis;
 using Microsoft.CodeAnalysis.Text;
 using Xunit;
+using Moq;
 
 namespace Microsoft.VisualStudio.ProjectSystem.VS
 {
     [ProjectSystemTrait]
     public class RenamerTests
     {
-        bool ConfirmForRenameWasCalled { get; set; }
+        [Theory]
+        [InlineData("class Foo{}", "Foo.cs", "Bar.cs")]
+        [InlineData("interface Foo { void m1(); void m2();}", "Foo.cs", "Bar.cs")]
+        [InlineData("delegate int Foo(string s);", "Foo.cs", "Bar.cs")]
+        [InlineData("partial class Foo {} partial class Foo {}", "Foo.cs", "Bar.cs")]
+        [InlineData("struct Foo { decimal price; string title; string author;}", "Foo.cs", "Bar.cs")]
+        [InlineData("enum Foo { None, enum1, enum2, enum3, enum4 };", "Foo.cs", "Bar.cs")]
+        [InlineData("namespace n1 {class Foo{}} namespace n2 {class Foo{}}", "Foo.cs", "Bar.cs")]
```

Comment:
The test cases in a single line is easy to understand for cases where there is a single level of nesting. For Eg: The first testcase, `class Foo{}`, is easy to understand. But it gets difficult to understand what the code looks like here `namespace n1 {class Foo{}} namespace n2 {class Foo{}}`. Not very difficult but if the code is written in the same way as we would write with line breaks, indents and everything, it will be very easy to understand. I understand that we haven't reached that high level of complexity for test code but this is something to keep in mind as we implement new feature and write more tests